x-pack/plugins/transform/public/app/sections/create_transform/components/step_define/hooks/use_agg.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { StepDefineExposedState } from '../common';
+
+export const useAgg = (defaults: StepDefineExposedState) => {};
```

Comment:
what is the purpose of this hook?

Reply:
Well spotted :) — Was a leftover from an earlier refactor, removed in b413505.